CuriousKatie/CuriousKatie/Model/Interests.swift

```diff
@@ -16,17 +16,40 @@ enum Skill: String {
 
 struct Interest: Equatable {
     
-    static func == (lhs: Interest, rhs: Interest) -> Bool {
-        return lhs.title == rhs.title
-        
-    }
-
     var title: String
     var skill: Skill
     
     init(title: String, skill: Skill) {
         self.title = title
         self.skill = skill
     }
+    
+    static func generatePossibleInterests() -> [Interest] {
```

Comment:
To when making an **explicit** (does not depend on any dynamic/parameter value that needs to be passed in) computation,
- Computed properties are better.
```suggestion
    static var possibleInterests: [Interest] {
```